test/fixtures/output/kafka/acl/create-help-onprem.golden

```diff
@@ -35,9 +35,9 @@ Flags:
       --prefix                    Set to match all resource names prefixed with this value.
       --url string                Base URL of REST Proxy Endpoint of Kafka Cluster (include "/kafka" for embedded Rest Proxy). Must set flag or CONFLUENT_REST_URL.
       --ca-cert-path string       Path to a PEM-encoded CA to verify the Confluent REST Proxy.
-      --client-cert-path string   Path to client cert to be verified by Confluent REST Proxy, include for mTLS authentication.
+      --client-cert-path string   Path to client cert to be verified by Confluent REST Proxy. Include for mTLS authentication.
       --client-key-path string    Path to client private key, include for mTLS authentication.
-      --no-authentication         Include if requests should be made without authentication headers, and user will not be prompted for credentials.
+      --no-authentication         Include if requests should be made without authentication headers and user will not be prompted for credentials.
       --prompt                    Bypass use of available login credentials and prompt for Kafka Rest credentials.
       --context string            CLI context name.
   -o, --output string             Specify the output format as "human", "json", or "yaml". (default "human")
```

Comment:
@tim-elhajj I didn't make the `default` to `Default` change because that's Cobra's formatting for adding flags with a default value.